test/e2e/specs/specs-wpcom/wp-editor__post-advanced-flow.ts

```diff
@@ -0,0 +1,162 @@
+/**
+ * @group gutenberg
+ * @group calypso-pr
+ */
+
+import {
+	DataHelper,
+	GutenbergEditorPage,
+	envVariables,
+	TestAccount,
+	PostsPage,
+	ParagraphBlock,
+	EditorSettingsSidebarComponent,
+	SnackbarNotificationComponent,
+} from '@automattic/calypso-e2e';
+import { Browser, Page } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( `Editor: Advanced Post Flow` ), function () {
+	const accountName = envVariables.GUTENBERG_EDGE
+		? 'gutenbergSimpleSiteEdgeUser'
+		: 'simpleSitePersonalPlanUser';
+	const postTitle = `Post Life Cycle: ${ DataHelper.getTimestamp() }`;
+	const originalContent = DataHelper.getRandomPhrase();
+	const additionalContent = 'Updated post content';
+
+	let page: Page;
+	let gutenbergEditorPage: GutenbergEditorPage;
+	let paragraphBlock: ParagraphBlock;
+	let postURL: string;
+
+	beforeAll( async function () {
+		page = await browser.newPage();
+
+		const testAccount = new TestAccount( accountName );
+		await testAccount.authenticate( page );
+
+		gutenbergEditorPage = new GutenbergEditorPage( page );
+		await gutenbergEditorPage.visit( 'post' );
```

Comment:
Also in the vein of "let's test more Calypso", what do you think of starting this test on the `/posts` page and clicking on the "Add new post" button? Might be a good validation that takes you into the editor correctly.

Reply:
Interesting call. I'll tinker with it.